lms/djangoapps/courseware/access.py

```diff
@@ -801,6 +801,33 @@ def _visible_to_nonstaff_users(descriptor):
     return VisibilityError() if descriptor.visible_to_staff_only else ACCESS_GRANTED
 
 
+def _user_passes_milestones_for_descriptor(descriptor, course_key, user):
```

Comment:
Can you change the signature of this function to match the signature of other functions in this file?  This includes the naming pattern as well as the order of the parameters.  So it would be something like:
`_can_access_descriptor_with_milestones(user, descriptor, course_key)`